bindings/rust/wasmedge-sdk/examples/hello.rs

```diff
@@ -2,33 +2,33 @@
 
```

Comment:
Because `wasmedge-sdk` crate will be dropped and may be redesigned in the future.  
This comment is acceptable in this PR.

Reply:
Thanks!

Reply:
We should add some comments/git commit message to explain the reason. It looks like just comment out the example without any reason in both commit history and comments.

Reply:
Accepted. Will add a comment for it. Thanks!